### PR TITLE
Ret 846

### PR DESCRIPTION
**Pull request by lvo700**
### JIRA link ###

https://tools.hmcts.net/jira/browse/RET-846

### Change description ###

Added the 'Acas Certificate for more than one respondent' page.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X ] No
```

### CODE REVIEW

File: src/test/unit/mocks/mockRequest.ts [hunk 1]
```diff
@@ -26,3 +26,29 @@ export const mockRequest = ({
   } as any;
   return req;
 };
+
+export const acasMultipleMockRequest = ({
```
Comment (tautvydas-cerniauskas) [inline src/test/unit/mocks/mockRequest.ts:30]: Why do you need a new mochRequest? Can you not just provide data into the existing one?

File: src/main/definitions/case.ts [hunk 2]
```diff
@@ -12,7 +12,12 @@ export interface CaseDate {
 }
 
 export interface Case {
-  dobDate: CaseDate
+  dobDate: CaseDate;
+  address1?: string;
```
Comment (marcjudge) [inline src/main/definitions/case.ts:16]: Can the whole address be null? Or do we need at least one non-nullable address field?

File: src/main/controllers/helpers.ts [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+            v.value = caseName[key];
+          }
+        });
+        return v;
+      });
+      for (const [, value] of Object.entries(fields)) {
+        (value as FormOptions)?.values
+          ?.filter((option: FormInput) => option.subFields !== undefined)
+          .map((fieldWithSubFields: FormInput) => fieldWithSubFields.subFields)
+          .forEach((subField: any) => assignFormData(caseName, subField));
```
Comment (marcjudge) [inline src/main/controllers/helpers.ts:93]: Is it possible to assign a type to the subfield here? In general you should avoid 'any' if possible

File: src/main/definitions/case.ts (hunk 2 above)
Comment (tautvydas-cerniauskas) [reply src/main/definitions/case.ts:16]: This was more part of my PR. For the `Case` to work we only need to have one field as mandatory and everything else can be optional.

**Review by marcjudge — APPROVED** [review]

File: src/test/unit/mocks/mockRequest.ts (hunk 1 above)
Comment (lvo700) [reply src/test/unit/mocks/mockRequest.ts]: Removed the extra mock request and amended the controller test file to include the radio buttons inside userCase constant.

**Review by tautvydas-cerniauskas — APPROVED** [review]

**Pull request merged**